preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -208,6 +208,14 @@ public void discardChanges() {
     preferencesFxModel.discardChanges();
   }
 
+  /**
+   * Check if any changes are pending. Useful when manually saving settings.
+   * @return true if there are unsaved changed.
+   */
+  public boolean areSettingsPending() {
```

Comment:
For methods returning a boolean please use a name that starts with "is". E.g.: isContainingChanges()

Reply:
@dlemmermann Change has been made.